packages/camera/camera_android_camerax/CONTRIBUTING.md

```diff
@@ -0,0 +1,46 @@
+# Contributing to camera\_android\_camerax
+
+## Note on the plugin structure
```

Comment:
nit:

I think it would be simpler to just call this section `Plugin Structure`